app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from os import path, chdir
+from sys import argv
+
+
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
```

Comment:
`" "` is default value for `split`.
```suggestion
    parts = command.split()
```

Reply:
`" "` is default value for `split()`